priv/repo/migrations/20170308222552_create_auth_token.exs

```diff
@@ -0,0 +1,14 @@
+defmodule CodeCorps.Repo.Migrations.CreateAuthToken do
+  use Ecto.Migration
+
+  def change do
+    create table(:auth_token) do
```

Comment:
Do we mean this to be a password reset token only? If so, should we call it that instead?

Also, I think you want the table name to be plural here.